README.md

```diff
@@ -1,94 +1,105 @@
+
+```shell
+sudo mkdir -m=777 /usr/local/src/blink-server
+git clone https://github.com/drpain/blink-server.git /usr/local/src/blink-server
+sudo ln -s /usr/local/src/blink-server/bluetooth_server.py /usr/local/bin/blink-server
 ```
-You also need to add --compat as described below.
 
 ## STARTING SERVER
 
 Then to start the server, all you need to do is run the server with the following command from your terminal. As long as the server runs, you will be able to send commands to it. 
 
-```
-sudo ~/.blink/bluetooth_server.py
+```shell
+sudo blink_server
 ```
 
 ## HELP
 
-The hardest part of this is getting the server installed and running. If you struggle to get the bluetooth to work, maybe this will help:
+The hardest part of this is getting the server installed and running. If you struggle to get the bluetooth to work, check these FAQ:
+
+### I can't connect my phone to my laptop
+
+From this [very useful guide](http://blog.davidvassallo.me/2014/05/11/android-linux-raspberry-pi-bluetooth-communication/):
 
-*This guide helped me allot, original post: [Bluetooth Setup](http://blog.davidvassallo.me/2014/05/11/android-linux-raspberry-pi-bluetooth-communication/)*
 > There are plenty of guides in the internet on how to get bluetooth working, but the only method that worked consistently for me is the following:
 > Disable bluetooth pnat support as there seems to be a bug which stops proper operation with pnat enabled. Full details can be found here:  
 > https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=690749
 
 > A workaround is to add the following to /etc/bluetooth/main.conf:
-```
+```config
 DisablePlugins = pnat
 ```
 
 
-### Permissions Issues:
-This has been reported as helpfull for sorting out permissions issues on Ubuntu 15.10. Thanks @jachym for providing the link:
-https://code.google.com/p/pybluez/issues/detail?id=62
-
-Something along the lines of:
-```bluetooth.btcommon.BluetoothError: (2, 'No such file or directory'```
+### `bluetooth.btcommon.BluetoothError: (2, 'No such file or directory')`
+This is a known issue on Ubuntu 15.04 and up, where we'll have to run the bluetooth daemon in **compat mode**. Thanks @jachym for providing a link with the [solution](https://code.google.com/p/pybluez/issues/detail?id=62):
 
 ```shell
-# https://stackoverflow.com/a/46810116
-# Adding compatibility mode to the service will help start it
-
+# Open the config file in your favorite editor
 sudo vim /etc/systemd/system/dbus-org.bluez.service
 
-# Find the line with the `ExecStart` and replace it  with
-ExecStart=/usr/lib/bluetooth/bluetoothd -C
+# Find the line with the `ExecStart` and replace it with
+ExecStart=/usr/lib/bluetooth/bluetoothd --compat
 
-# And complete the action with the following commands
-sudo sdptool add SP
+# Restart the bluetooth daemon
+sudo sdptool add SP  # This command isn't necessary on Arch Linux
 sudo systemctl daemon-reload
-sudo service bluetooth restart
+sudo systemctl restart bluetooth
```

Comment:
This may be specific to systems, whereas most will support the service comment.

Reply:
What about line 82 then? Is `systemctl daemon-reload` needed if you use `service`? I've only ever used `systemctl` myself so I don't know.

Reply:
I can't remember ever using Arch, but I do use Debian based / REHL based systems.  Since they started using systemd, you use the services command for them. systemctl in these systems is only used to change the startup for a service, and to enable / disable services at startup.

Ok yes, if you don't mind updating it to indicate both, since it would depend on the linux distro you are running.